gapis/server/server.go

```diff
@@ -113,29 +113,36 @@ func (s *server) GetServerInfo(ctx context.Context) (*service.ServerInfo, error)
 	return s.info, nil
 }
 
-func (s *server) CheckForUpdates(ctx context.Context, includePrereleases bool) (*service.Release, error) {
+func (s *server) CheckForUpdates(ctx context.Context, includeDevReleases bool) (*service.Release, error) {
 	const (
-		githubOrg  = "google"
-		githubRepo = "gapid"
+		githubOrg     = "google"
+		githubRepo    = "gapid"
+		devGithubRepo = "gapid-dev-releases"
 	)
 	ctx = status.Start(ctx, "RPC CheckForUpdates")
 	defer status.Finish(ctx)
 	ctx = log.Enter(ctx, "CheckForUpdates")
+
 	client := github.NewClient(nil)
 	options := &github.ListOptions{}
 	releases, _, err := client.Repositories.ListReleases(ctx, githubOrg, githubRepo, options)
 	if err != nil {
 		return nil, log.Err(ctx, err, "Failed to list releases")
 	}
+
+	if includeDevReleases {
+		devReleases, _, err := client.Repositories.ListReleases(ctx, githubOrg, devGithubRepo, options)
+		if err != nil {
+			return nil, log.Err(ctx, err, "Failed to list dev-releases")
+		}
+		releases = append(releases, devReleases...)
+	}
+
 	var mostRecent *service.Release
 	mostRecentVersion := app.Version
 	mostRecentDevVersion := app.Version.GetDevVersion()
 
 	for _, release := range releases {
-		if !includePrereleases && release.GetPrerelease() {
-			continue
-		}
-
```

Comment:
If you remove this filtering here, you're going to have to make the logic below smarter to filter dev releases when `includeDevReleases` is false.

Reply:
Sorry I forgot to say: the code now assumes all dev-releases will be on the new repo. So the `includeDevReleases` boolean now means: do list dev-releases from the new repo (whereas includePrerelease filtered on the GitHub notion of a pre-release).

Then the logic to select the most recent is oblivious to whether we want dev-releases or not.

I will take care of cleaning up the release page of the main repo before this change is merged in.

How does that sound?

Reply:
Yes, I understand that. However, we then need to do either
1) always filter pre-releases
2) add another boolean to indicate whether to include pre-releases (and set it to false in the UI)

With this change as is, pre-releases are treated as regular releases and would cause a notification. This is different behavior than before where we were either ignoring them or users had to explicitly opt-in to be notified of pre-releases.